doc/source/user_guide/temperature.rst

```diff
@@ -1,8 +1,8 @@
 .. _temperature:
 
-===================================
-Temperature & Change of Temperature
-===================================
+==================================
+Temperature and temperature change
```

Comment:
This is an improvement, although difference rather than change is definitely the norm.